krummstab/__init__.py

```diff
@@ -163,7 +157,7 @@ def move_content_and_delete(src: pathlib.Path, dst: pathlib.Path) -> None:
 # ============================== Summarize Sub-Command =========================
 
 
-def summarize(_the_config: config.Config) -> None:
+def summarize(_the_config: config.Config, args) -> None:
```

Comment:
Would be nice to add type information for the new parameter in case it is informative and not too verbose. This applies to all sub-command definitions.